core/src/main/resources/i18n/displayStrings.properties

```diff
@@ -116,6 +116,8 @@ shared.OR=OR
 shared.notEnoughFunds=You don''t have enough funds in your Haveno wallet for this transaction—{0} is needed but only {1} is available.\n\nPlease add funds from an external wallet, or fund your Haveno wallet at Funds > Receive Funds.
 shared.waitingForFunds=Waiting for funds...
 shared.depositTransactionId=Deposit transaction ID
```

Comment:
I was expecting `shared.yourDepositTransactionId`, `shared.peerDepositTransactionId`, `shared.makerDepositTransactionId` and `shared.takerDepositTransactionId` based on our conversation.

Reply:
Removed `depositTransactionId` completely and added taker/maker.